aws/resource_aws_chime_voice_connector_test.go

```diff
@@ -0,0 +1,96 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/chime"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAWSChimeVoiceConnector_basic(t *testing.T) {
```

Comment:
There should also be a test for updating the resource and for testing when a resource disappears. Looking at one of the other resources with an `_update` and `_disappears` test should provide some good examples.